src/screen/HomeScreen.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from 'react';
+import {
+  ScrollView, FlatList, RefreshControl
+} from 'react-native';
+import { connect } from 'react-redux';
+import { Button } from 'react-native-elements';
+import ListItem  from '../components/ListItem';
+import Block from '../components/Block';
+import {
+  getPosts, rmPost, savePost, textEdit
+} from '../actions/PostActions';
+
+
+const HomeScreen = ({
+  posts,
+  getPostsAction,
+  navigation,
+}) => {
+  useEffect(() => {
+    getPostsAction();
+  }, []);
+
+  const [isLoading, setLoading] = useState(false);
+
+  const update = async () => {
+    setLoading(false);
+
+    posts = getPostsAction();
+  };
+
+  const renderItem = ({ item }) => (
+    <ListItem
+      title={item.title}
+      text={item.text}
+      onPress={() => {
+        navigation.navigate('PostScreen', {
```

Comment:
Why you ignore my comments, i said that you should create separate function for navigation? Avoid using arrow-functions in onPress